components/FeaturedTools/FeaturedTools.css.ts

```diff
@@ -41,7 +41,7 @@ export const FeaturedToolDescriptionWrapper = styled.div`
   display: flex;
   width: 40%;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: center;
```

Comment:
The reason I did `space-between` rather than center was so that the button would be all the way at the bottom and the header would be at the top aligned with the header and button for the featured builds on smaller screens. With this change, the button doesn't line up with the other one.